Makefile

```diff
@@ -20,5 +20,6 @@ clean:
 	rm -rf android/libs
 	rm -rf ios/build
 	rm -rf ios/dist
+	rm -rf ~/Library/Application Support/Titanium/modules/iphone/ti.airship/
```

Comment:
I'm not sure I remember why, but I recall that using `${HOME}` is better than `~`. Maybe something to do with different shells?